jfsd/utils.py

```diff
@@ -375,16 +294,238 @@ def initialize_single_neighborlist(
     neighbor_fn = partition.neighbor_list(displacement,  
                                               jnp.array([Lx, Ly, Lz]),
                                               r_cutoff=space_cut,  # Spatial cutoff for 2 particles to be neighbor
-                                              dr_threshold=0.1,  # displacement of particles threshold to recompute neighbor list
-                                              capacity_multiplier=1,
+                                              #dr_threshold=0.1,  # displacement of particles threshold to recompute neighbor list
+                                              capacity_multiplier=1.1,
```

Comment:
Why did it change and why should it be 1.1?

Reply:
The size of the allocated neighbor list depends on the capacity multiplier. If it's 1, then the list will have the smallest size needed to accommodate all neighbors. This means that once the particle configuration change to a one where the list is not big enough, we need to reallocate it, and this cannot be done on GPU. Thus, to reduce the amount of time we re-allocate the lists, we increase a bit the initial size of the lists.